src/iotjs_binding.cpp

```diff
@@ -70,34 +70,32 @@ JObject::JObject(const JObject& other) {
 
 
 JObject::JObject(bool v) {
-  _obj_val = JVal::Bool(v);
+  _obj_val = iotjs_jval_bool(v);
```

Comment:
But I also think that there might be some redundancy here or somewhere.
I think old c++ binding layer had 2 purposes in general: 1. support C++ language, and 2. make binding easier(memory mgnt, etc). Since we are changing our code into c, if some code is written only for the first purpose, it should be removed. Please let me know if there are something more to remove.